content/en/events/2023/kcscn/schedule.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Schedule
+type: docs
+weight: 3
+---
+
+**NOTE:** You **MUST** pick up your badge before attending the summit event.
+
+## CFP
+
+If you would like to submit a formal session, please use the [CFP Form](TODO).
+This form will be open from Monday, August 14th, and covers presentation, and collaboration style sessions.
+
+## Unconference
+
+Unconference sessions should use the GitHub issue <https://github.com/kubernetes/community/issues/TODO>.
+The unconference session will start after several talks. If you have a topic in mind, please add it to the Github Issue.
+
+## Schedule
+
+{{% schedule
+  text="Kubernetes Contributor Summit China 2023"
+  link="TODO"
+  script="TODO"
+%}}
```

Comment:
```suggestion
Schedule coming soon.
```

Similar thing here, we can remove this block for now just to get the page live and update it when we have links.

Reply:
Okay. Removed.